buildSrc/src/main/rocker/com/uber/okbuck/template/jvm/JvmRule.rocker.raw

```diff
@@ -62,7 +63,14 @@ lib_deps = []
     }
     ],
 }
-@if (valid(annotationProcessors)) {
+@if (valid(annotationProcessors) && useAnnotationProcessorPlugin) {
```

Comment:
It feels a bit weird to leak an experimental flag to the rocker template. This should be business logic contained in the composer

Reply:
discussed offline, the experimental flag is just a temporary field which we will remove on the next major release after this, so fine to keep it for now. 
Also moving this logic to composer would require modifying bunch of composers instead putting the logic in the rocker template.